src/index.css

```diff
@@ -1,13 +1,9 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@import url('https://fonts.googleapis.com/css2?family=Secular+One&display=swap');
```

Comment:
- [ ] The sign up page needs a little more styling.  Also, the login page:

![image](https://user-images.githubusercontent.com/57421823/227429678-5d1cbf6a-db84-452f-8ae0-220e0e671584.png)


I'm pretty sure this is a work in progress, and it's going to be awesome. 💪